src/App.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import NavBar from "./components/navbar";
```

Comment:
Try absolute imports for this project 

https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8

Reply:
done